pkg/kernel/kernel_test.go

```diff
@@ -39,6 +40,7 @@ func TestSetAffineAttributes(t *testing.T) {
 			obj := newObj(kind, objNewName)
 
 			err := SetAffineAttributes(obj, kernelFullVersion, operatingSystemMajorMinor)
+			require.NoError(t, err)
```

Comment:
No familiar with that code, can you explain what this method is supposed to do and why it is fixing the CI?

Reply:
This is a unit test that was introduced in #121. We were initializing `err` to the result of `SetAffineAttributes(obj, kernelFullVersion, operatingSystemMajorMinor)`, but never actually checking the contents of `err`.  
This PR fixes this by making sure that `err` is not an error, using [require.NoError](https://pkg.go.dev/github.com/stretchr/testify/require#NoError).